spec/data-types-and-values.html

```diff
@@ -159,8 +159,7 @@ <h1>
 
     <emu-clause id="sec-ecmascript-language-types-box-type">
       <h1>The Box Type</h1>
-      <p>The Box type is the set of all the possible singleton primitive wrappers around any ECMAScript value. Each box value holds an associated [[Value]] containing an ECMAScript value. The [[Value]] internal slot is never modified.</p>
-
+      <p>The Box type is the set of all the possible singleton primitive wrappers around any ECMAScript value. Each box value holds an associated [[Value]] containing an Object. The [[Value]] internal slot is never modified.</p>

```

Comment:
Should we also mention the `[[Realm]]` internal slot here? And remove the 'any ECMAScript value' ?

I think for every object and realm pair, there is a unique corresponding Box value. But not sure how to say that in emca-speak.